auth-web/src/views/auth/home/MaintainBusinessView.vue

```diff
@@ -24,17 +24,18 @@
         </v-list>
         <!-- Panel Btns -->
         <div class="incorporate-btns">
-          <v-btn v-if="userProfile" large color="bcgovgold" class="cta-btn font-weight-bold mr-3" @click="emitManageBusinesses()">
-            Manage an Existing Business
+          <v-btn large color="bcgovblue" class="cta-btn font-weight-bold mr-2 white--text"
+            @click="emitRedirectManage()">
+            Manage my Business
           </v-btn>
-          <template v-else>
-            <v-btn large color="bcgovgold" class="cta-btn font-weight-bold mr-3" to="/choose-authentication-method">
-              Create a BC Registries account
-            </v-btn>
-          </template>
-          <learn-more-button
-           :redirect-url="learnMoreUrl"
-          />
+          <learn-more-button :redirect-url="learnMoreUrl" :isWide="true"/>
+          <div d-flex class="mt-8">
```

Comment:
d-flex will be inside class?

Reply:
This is the same as in [here](https://github.com/bcgov/sbc-auth/blob/5033b678f797002aa6178bd738ad0757d20be53e/auth-web/src/views/auth/home/HomeView.vue#L52-L58). Do you want to me to put d-flex inside class like `class="d-flex mt-8"`?

Reply:
If you mean to use the d-flex then it should be in the class string. Otherwise you can remove it (both places please). Test to verify.

Reply:
Did you verify it?

Reply:
Yes. It's working well.